JeMPI_Apps/JeMPI_UI/src/components/browseRecords/BrowseRecords.tsx

```diff
@@ -51,15 +62,15 @@ const Records = () => {
   const navigate = useNavigate()
   const { apiClient } = useConfig()
   const { getFieldsByGroup } = useAppConfig()
-
+  const [startDateFilter, setStartDateFilter] = useState<Dayjs>(
+    dayjs().startOf('day')
+  )
+  const [endDateFilter, setEndDateFilter] = useState<Dayjs>(
+    dayjs().endOf('day')
+  )
   const [searchQuery, setSearchQuery] = useState<Array<SearchParameter>>([])
-
-  const [dateFilter, setDateFilter] = useState(dayjs())
-
   const [dateSearch, setDateSearch] = useState(dayjs())
-
   const [searchParams, setSearchParams] = useSearchParams()
-
   const [isFetchingInteractions, setIsFetchingInteractions] = useState(
     searchParams.get('isFetchingInteractions')
       ? JSON.parse(searchParams.get('isFetchingInteractions') as string)
```

Comment:
Does this not result in an ambigous type i.e `string` or `boolean`. Why can't we convert this `isFetchingInteractions` to a boolean?